docs/dmytro-shevchenko1/GamesProject/calculator.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" type="text/css" href="./styles/style.css">
+        <link rel="stylesheet" type="text/css" href="./styles/header.css">
+        <link rel="stylesheet" type="text/css" href="./styles/calculator.css">
+        <link rel="stylesheet" type="text/css" href="styles/footer.css">
+        <title>Calculator</title>
+    </head>
+    <body class="calculator-page">
+        <header class="header">
+            <div class="container">
+                <div class="header-flex">
+                    <img src="images/logo.svg" alt="Logo">
+                    <div class="nav">
+                        <ul class="nav-items">
+                            <li>
+                                <a class="calc-link" href="calculator.html">Calculator</a>
+                            </li>
+                            <li>
+                                <a class="cards-link" href="cards.html">Games</a>
+                            </li>
+                            <li>
+                                <a class="about-link" href="index.html">About</a>
+                            </li>
+                            <li>
+                                <a class="login-link" href="login.html">Login</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <main class="calculator-field">
+            <h1 class="calculator-h1">Calculator v. 1</h1>
+            <div class="field">
+                <label>
+                    <input id="firstValue">
+                </label>
+                <label>
+                    <select class="select-operations" name="operation" id="operation">
+                        <option value="+">+</option>
+                        <option value="-">-</option>
+                        <option value="/">/</option>
+                        <option value="*">*</option>
+                        <option value="**">**</option>
+                    </select>
+                </label>
+                <label>
+                    <input id="secondValue">
+                </label>
+                <button class="result-button" id="calculate" type="button">=</button>
+                <span class="result-text" id="result"></span>
+            </div>
+        </main>
+        <footer class="footer">
```

Comment:
Don't you like to make your footer [sticky](https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Sticky_footers)?
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/f9a1fced-8e37-4a1d-9383-a1b4b5f7f58c)